lumicks/pylake/detail/image.py

```diff
@@ -93,7 +111,7 @@ def reconstruct_image(data, infowave, pixels_per_line, lines_per_frame=None, red
     data : array_like
         Raw data to use for the reconstruction. E.g. photon counts or force samples.
     infowave : array_like
-        The famous infowave.
+        The infamous infowave.
```

Comment:
(:

Reply:
Not the change we needed, but the change we deserved. :bat:

Reply:
😆